nix/libvirtd.nix

```diff
@@ -45,11 +45,19 @@ in
   ###### interface
 
   options = {
-    deployment.libvirtd.imageDir = mkOption {
-      type = types.path;
-      default = "/var/lib/libvirt/images";
+    deployment.libvirtd.storagePool = mkOption {
+      type = types.str;
+      default = "default";
+      description = ''
+        The storage pool where the virtual disk is be created.
```

Comment:
"is be"

Reply:
is it possible to add an example ?

Reply:
Thanks, fixed. :)

Reply:
Not sure about the example here. This option is just the name of the libvirt storage pool. By default, there is a single storage pool named "default" which points to the /var/lib/libvirt/images directory. It is really hard for me to come up with a meaningful example here other than "default" here. :) But we already use "default" as the default value.

Reply:
I see. I was not familiar with the terminology so this is the name of the storage pool ty

Reply:
Edited the description, should be more clear this way.